docker-compose.yml

```diff
@@ -26,3 +26,7 @@ services:
       - TEST_AZURE_ACCOUNT_NAME
       - TEST_AZURE_ACCOUNT_KEY
       - TEST_AZURE_CONTAINER_NAME
+      - TEST_GCP_STORAGE_API_URL
+      - TEST_GCP_STORAGE_API_TOKEN
+      - TEST_GCP_BUCKET
+      - TEST_GCP_SERVICE_ACCOUNT
```

Comment:
v `tests/bootstrap.php` se validuje jestli jsou zadany envy, tyhle tam chybi :)

plus kdyz uz nic jinyho, tak by bylo dobre je pridat i Readme.

Provisioning tech resourcu jsme se dohodli, ze udelam jako samostatne issue. Ted je to na pytel, melo by to byt pres terrbackupSignedUrlsaform.

Reply:
jo do readme jsem to doplnil